eclipse-che/che-operator
# Read trusted CA configmaps only from installation namespace

Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

### What does this PR do?
Limits searching namespace for trusted CA configmaps to only Che installation namespace

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/20564

### How to test this PR?
1. Deploy Eclipse Che
2. Create two configmaps with trusted CA certs: one in the installation namespace, he other one in another namespace
3. Check, that only CA certs from the first configmap is applied.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
